server/src/bots/telegram/services/messageRegistry.ts

```diff
@@ -56,4 +64,36 @@ class MessageRegistry {
 
 export const registry = new MessageRegistry();
 
-type CallBackQueryHandler = (bot: any, message: any, chatId: unknown) => any
+type CallBackQueryHandler = (bot: any, message: any, chatId: unknown, argument?: any) => any
+
+export function WithCommandArgument(handler: CallBackQueryHandler): any {
```

Comment:
I do not see point of changing of this in your PR as I have story for that which is already on PR since yesterday and there I am taking care of that